class/person.rb

```diff
@@ -1,10 +1,14 @@
 require 'securerandom'
+require '../nameable'
+require '../trimmer_decorator'
+require '../capitalize_decorator'
```

Comment:
Please use `require_relative` to import local files as `require` is used to import gems and external fil
![person](https://user-images.githubusercontent.com/35504149/180008210-80e01d8f-1b9b-4188-a288-4c55eef0f94e.png)
es.

![trimdecorator](https://user-images.githubusercontent.com/35504149/180008267-2b9fe3f7-f96b-4ab6-8dd0-7f09d1690ddf.png)
![capitaldecorator](https://user-images.githubusercontent.com/35504149/180008302-c39aabce-2406-4e66-a020-7d7c4cdf7d44.png)

Reply:
Thanks, @SajjadAhmad14  didn't know that detail 🤝